src/frontend/js/pages/DashboardCourses/_styles.scss

```
@@ -10,6 +10,21 @@
   }
 }
 
+.dashboard__list--loading {
+  &::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
```

My new preferred shortcut 😛 
```suggestion
    inset: 0;
```